src/routes/_components/status/Status.html

```diff
@@ -194,7 +194,7 @@
       originalAccount: ({ originalStatus }) => originalStatus.account,
       originalAccountId: ({ originalAccount }) => originalAccount.id,
       visibility: ({ originalStatus }) => originalStatus.visibility,
-      spoilerText: ({ originalStatus }) => originalStatus.spoiler_text,
+      spoilerText: ({ originalStatus }) => originalStatus.spoiler_text || ((originalStatus.content || '').length > 1024 ? "LONG" : undefined),
```

Comment:
I would like both `1024` and `"LONG"` to be pulled out as constants somewhere. Also wondering if instead of "LONG" it could show the number of characters or something.

Reply:
On second thought, let's just make it "Long"